um, caja is at 1.5 so 2.22 and older version probably is wrong

last gnome2 was 2.32.2.1 for nautilus
--- caja-dropbox-0.7.1/src/dropbox.c~	2012-07-16 19:04:32.000000000 +0300
+++ caja-dropbox-0.7.1/src/dropbox.c	2013-02-23 00:17:01.435618438 +0200
@@ -41,9 +41,7 @@
   caja_dropbox_register_type (module);
   type_list[0] = CAJA_TYPE_DROPBOX;
 
-  dropbox_use_caja_submenu_workaround
-    = (CAJA_VERSION_MAJOR < 2 ||
-       (CAJA_VERSION_MAJOR == 2 && CAJA_VERSION_MINOR <= 22));
+  dropbox_use_caja_submenu_workaround = FALSE;
   dropbox_use_operation_in_progress_workaround = TRUE;
 }
 
